test/e2e/tests/user-profile.spec.ts

```diff
@@ -91,12 +91,33 @@ test.describe('User Profile', () => {
     await userProfilePage.aboutMeTab.ageField.fill(newAge);
     await userProfilePage.aboutMeTab.genderField.selectOption({ label: 'Female' });
 
-    await userProfilePage.save();
+    await userProfilePage.saveAboutMe();
     await userProfilePage.reload();
 
     await expect(userProfilePage.aboutMeTab.nameField).toHaveValue(newName);
     await expect(userProfilePage.aboutMeTab.ageField).toHaveValue(newAge);
     await expect(userProfilePage.aboutMeTab.genderField).toHaveSelectedOption({ label: 'Female' });
   });
 
+  test('Delete user account', async ({ tab, userService }) => {
+    const user = await userService.createRandomUser();
+    await login(tab, user);
+
+    const userProfilePage = new UserProfilePage(tab);
+    await userProfilePage.goto();
+    await userProfilePage.tabs.deleteAccount.click();
+
+    const confirmation = 'DELETE';
+    await userProfilePage.deleteTab.confirmField.fill(confirmation);
+
+    const [loginPage] = await Promise.all([
+      LoginPage.waitFor(tab),
+      userProfilePage.deleteMyAccount(),
+    ]);
+
+    await loginPage.goto();
+
```

Comment:
```suggestion
```

Reply:
Sorry, I don't know why I didn't put my comments in a reivew 🙈

Reply:
Can you explain? I was thinking to make sure that if a user deletes their account, they are logged out (as they should be).